Formula/gcc.rb

```diff
@@ -64,17 +61,17 @@ def install
     ENV.delete "LD"
 
     # We avoiding building:
-    #  - Ada, which requires a pre-existing GCC Ada compiler to bootstrap
+    #  - Ada and D, which require a pre-existing GCC Ada compiler to bootstrap
     #  - Go, currently not supported on macOS
     #  - BRIG
     languages = %w[c c++ objc obj-c++ fortran]
-    languages << "d" if Hardware::CPU.intel?
 
     pkgversion = "Homebrew GCC #{pkg_version} #{build.used_options*" "}".strip
 
+    # Use `lib/gcc/current` to provide a path that doesn't change with GCC's version.
     args = %W[
       --prefix=#{opt_prefix}
-      --libdir=#{opt_lib}/gcc/#{version_suffix}
+      --libdir=#{opt_lib}/gcc/current
```

Comment:
CC @fxcoudert, in case this is not a good idea.

Reply:
I think it's a good idea, and I cannot see any drawback. But, this is a subtle issue, so I might be overlooking something 😄

Reply:
Yes, it's the subtle issues that I'm concerned about. Ah well, hopefully nothing 💥s